src/main/java/com/dev/cinema/controller/CinemaHallController.java

```diff
@@ -0,0 +1,38 @@
+package com.dev.cinema.controller;
+
+import com.dev.cinema.model.CinemaHall;
+import com.dev.cinema.model.dto.cinemahall.CinemaHallRequestDto;
+import com.dev.cinema.model.dto.cinemahall.CinemaHallResponseDto;
+import com.dev.cinema.model.dto.mapperdto.CinemaHallMapper;
+import com.dev.cinema.service.CinemaHallService;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema-hall")
+public class CinemaHallController {
+    @Autowired
+    private CinemaHallService cinemaHallService;
+    @Autowired
+    private CinemaHallMapper cinemaHallMapper;
+
+    @PostMapping("/add")
```

Comment:
```suggestion
    @PostMapping
```
додатковий add - це зайве